aws/arctic-wolf/arctic_wolf.tf

```diff
@@ -55,6 +55,30 @@ variable "cloudtrail_name" {
   description = ""
 }
 
+variable "waf_arn" {
+  type        = map
+  default     = null
+  description = ""
+}
+
+variable "log_bucket_id" {
```

Comment:
change to `cloudtrail_bucket_id` if logging is specific to cloudtrail

Reply:
Updated variable naming